api/src/main/java/org/apache/cloudstack/api/command/user/network/RestartNetworkCmd.java

```diff
@@ -59,6 +59,9 @@
     @Parameter(name = ApiConstants.MAKEREDUNDANT, type = CommandType.BOOLEAN, required = false, description = "Turn the network into a network with redundant routers.", since = "4.11.1")
     private Boolean makeRedundant = false;
 
+    @Parameter(name = ApiConstants.LIVE_PATCH, type = CommandType.BOOLEAN, required = false, description = "Live Patch the router before restarting it. This parameter will work only when 'cleanup' is false.", since = "4.16.1")
```

Comment:
Can you explain it, since this will be consumed by users, for example:
```
Live patches the router software without restarting the network virtual router. This parameter will work only when 'cleanup' is false."
```
** also check if you meant `without restarting the network VR`?
Fix since version also to 4.17.